ui-e2e/envs/contracts.env

```diff
@@ -32,3 +32,5 @@ FOREIGN_GAS_PRICE=10000000000
 
 REQUIRED_NUMBER_OF_VALIDATORS=1
 VALIDATORS="0xaaB52d66283F7A1D5978bcFcB55721ACB467384b"
+
+DEPLOYMENT_GAS_LIMIT=5000000
```

Comment:
you already have this parameter in the line 4 but with a different value.

Reply:
Done.